services/storage/store.go

```diff
@@ -293,16 +344,6 @@ func (s *Store) TagKeys(ctx context.Context, req *datatypes.TagKeysRequest) (cur
 }
 
 func (s *Store) TagValues(ctx context.Context, req *datatypes.TagValuesRequest) (cursors.StringIterator, error) {
-	if tagKey, ok := measurementRemap[req.TagKey]; ok {
```

Comment:
This looks like a good optimisation to keep?

Reply:
I think it's added below, starting here: https://github.com/influxdata/influxdb/pull/21160/files#diff-6d1e664d01750fd2f45528ac06c5a98288a4d14b20e5fe3b36b759eedc8b4cc9R387